lib/xgu/nv2a_regs.h

```diff
@@ -0,0 +1,1321 @@
+#       define NV097_SET_SURFACE_PITCH_ZETA                       0xFFFF0000
+#   define NV097_SET_SURFACE_COLOR_OFFSET                     0x00000210
+#   define NV097_SET_SURFACE_ZETA_OFFSET                      0x00000214
+#   define NV097_SET_COMBINER_ALPHA_ICW                       0x00000260
+#   define NV097_SET_COMBINER_SPECULAR_FOG_CW0                0x00000288
+#   define NV097_SET_COMBINER_SPECULAR_FOG_CW1                0x0000028C
+#   define NV097_SET_CONTROL0                                 0x00000290
+#       define NV097_SET_CONTROL0_STENCIL_WRITE_ENABLE            (1 << 0)
+#       define NV097_SET_CONTROL0_Z_FORMAT                        (1 << 12)
+#       define NV097_SET_CONTROL0_Z_PERSPECTIVE_ENABLE            (1 << 16)
+#   define NV097_SET_FOG_MODE                                 0x0000029C
+#       define NV097_SET_FOG_MODE_V_LINEAR                        0x2601
+#       define NV097_SET_FOG_MODE_V_EXP                           0x800
+#       define NV097_SET_FOG_MODE_V_EXP2                          0x801
+#       define NV097_SET_FOG_MODE_V_EXP_ABS                       0x802
+#       define NV097_SET_FOG_MODE_V_EXP2_ABS                      0x803
+#       define NV097_SET_FOG_MODE_V_LINEAR_ABS                    0x804
+#   define NV097_SET_FOG_GEN_MODE                             0x000002A0
+#       define NV097_SET_FOG_GEN_MODE_V_SPEC_ALPHA                0
+#       define NV097_SET_FOG_GEN_MODE_V_RADIAL                    1
+#       define NV097_SET_FOG_GEN_MODE_V_PLANAR                    2
+#       define NV097_SET_FOG_GEN_MODE_V_ABS_PLANAR                3
+#       define NV097_SET_FOG_GEN_MODE_V_FOG_X                     6
+#   define NV097_SET_FOG_ENABLE                               0x000002A4
+#   define NV097_SET_FOG_COLOR                                0x000002A8
+#       define NV097_SET_FOG_COLOR_RED                            0x000000FF
+#       define NV097_SET_FOG_COLOR_GREEN                          0x0000FF00
+#       define NV097_SET_FOG_COLOR_BLUE                           0x00FF0000
+#       define NV097_SET_FOG_COLOR_ALPHA                          0xFF000000
+#   define NV097_SET_WINDOW_CLIP_TYPE                         0x000002B4
+#   define NV097_SET_WINDOW_CLIP_HORIZONTAL                   0x000002C0
+#       define NV097_SET_WINDOW_CLIP_HORIZONTAL_XMIN              0x00000FFF
+#       define NV097_SET_WINDOW_CLIP_HORIZONTAL_XMAX              0x0FFF0000
+#   define NV097_SET_WINDOW_CLIP_VERTICAL                     0x000002E0
+#   define NV097_SET_ALPHA_TEST_ENABLE                        0x00000300
+#   define NV097_SET_BLEND_ENABLE                             0x00000304
+#   define NV097_SET_CULL_FACE_ENABLE                         0x00000308
+#   define NV097_SET_DEPTH_TEST_ENABLE                        0x0000030C
+#   define NV097_SET_DITHER_ENABLE                            0x00000310
+#   define NV097_SET_LIGHTING_ENABLE                          0x00000314
```

Comment:
This and above `_ENABLE` stuff is needed (will mostly be boolean parameters; see existing `alpha_func` handler)